src/main/java/org/codehaus/mojo/sql/SqlExecMojo.java

```diff
@@ -256,6 +256,15 @@
     @Parameter( defaultValue = "false", property = "autocommit" )
     private boolean autocommit;
 
+    ////////////////////////////// Operation Configuration ////////////////////
+    /**
+     * Set to <code>true</code> to execute immediately rollback transactional SQL.
+     *
+     * @since 1.0
```

Comment:
This means this feature (parameter) is available since version 3.0.0 of the plugin...I already fixed this..